src/_includes/head.html

```diff
@@ -27,7 +27,7 @@
   {% assign og_image_path = page.image | default: layout.image | default: '/images/flutter-logo-sharing.png' %}
   <meta property="og:image" content="{{og_image_path | absolute_url}}">
 
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Extended" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Are we sure we don't need the Extended set? I _think_ we don't, but I'd rather err on the side of caution.

Reply:
AFAIK, this is how one might use an icon from the extended collection:

- Reference the font family (`font-family: 'Material Icons Extended';`) directly, or
- Use the CSS class `.material-icons-extended`, or
- `<span class="mdi mdi-file-upload-outline"></span>`, or
- Use a direct link: https://petershaggynoble.github.io/MDI-Sandbox/?icon=file-upload-outline, or
- Use a local image of the icon, but in that case we don't need to load the font.

(If you can think of other ways, let me know.)

The site source contains no such uses.

Reply:
Ah, thanks for the due diligence on this.